doc/security.md

```diff
@@ -162,3 +162,7 @@ On install, add the following arguments to the helm command:
 ```
 --set operator.controller.luksSecret=linstor-pass
 ```
+
+| :warning: WARNING                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| If you ever delete or change the passphrase secret, the LINSTOR Controller can no longer start with a failure message like `Automatic injection of passphrase failed`. You can force the Controller to start by setting the `luksSecret` value in the `LinstorController` resource to `""`. This will _not_ give you access to encrypted items such as remotes, but it will allow the Controller to start. If you need to recover encrypted values, you need to restore the original secret. |
```

Comment:
What about the case when original secret was lost? Is there any opportunity to assign the new one?

Reply:
Just confirmed by the linstor team: no, you can't assign a new one without knowing the old one.

Reply:
So in that case the linstor-cluster must be fully recreated with losing configuration of other (maybe unencrypted devices) 🤔